cabal.project

```diff
@@ -302,99 +302,113 @@ source-repository-package
   subdir: ouroboros-network
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
   subdir: io-sim
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
   subdir: ouroboros-network-testing
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
   subdir: ouroboros-consensus
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
+  subdir: ouroboros-consensus/ouroboros-consensus-test-infra
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/ouroboros-network
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
   subdir: ouroboros-consensus/ouroboros-consensus-mock
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
   subdir: ouroboros-consensus-byron
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
   subdir: ouroboros-consensus-shelley
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 9d9754c9ddcfff82b27c371a545aa4680d86d996
-  --sha256: 18ws841jn6hhmm3pqd22lmy20cgnp430dk3s07jzw3d5bpf3i34v
+  tag: 0143c516f860a2c790911bc4cc177569d7d04710
+  --sha256: 0608rashhvybqd1h6fjq9bvnzbyywhxbjxxr9h55m5fjp7i2v3sz
+  subdir: ouroboros-consensus-shelley-test
```

Comment:
Why is this needed?

Maybe you need to disable some `ouroboros-consensus*` testsuites in this file?